srcs/close.c

```diff
@@ -39,3 +45,12 @@ void	exit_error(char *msg)
 {
 	exit(error_msg(msg));
 }
+
+int	close_game(t_info *info)
+{
+	free_map(info->map);
+	destroy_window(info);
+	destroy_images(info);
+	exit(0);
+	return (0);
```

Comment:
Because this function is called by this hook: (click on the cross event)
`mlx_hook(info->mlx[WINDOW], 17, 0L << 0, close_game, info);`
and those functions prototypes require an int as return value.
It will never be catch though